frontend/src/app/pages/marker-page/marker-page.component.html

```diff
@@ -66,7 +66,7 @@ <h3>Actions</h3>
                 <button mat-raised-button color="primary" (click)="skipScan()">
                     Skip scan
                 </button>
-                <button mat-raised-button color="accent" (click)="sendCompleteLabel()"
+                <button mat-raised-button color="accent" [disabled]="!canSendScan()" (click)="sendCompleteLabel()"
```

Comment:
What if user won't click "Stop" and move to other Slice or switch Tool? :)

As an end user I may be a little bit confused seeing disabled button without any reason. Maybe a tooltip saying "You've got to end your Chain label first!" would be nice.